### DISCUSSION TITLE
Please continue to support manual installs for Teslamate!

### DISCUSSION THREAD

**Discussion by jensb** [teslamate-org/teslamate#3516] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

As per comment in PR#3439, I am repeating my request to continue to support manual installs here.
I am running Teslamate on a Raspberry Pi which already has instances of Grafana, Postgresql and a MQTT service running for a lot of services.
Duplicating all these for each service would totally overload the little machine and - IMHO - makes no sense.

So I would continue to need a supported way of installing either a *single* Teslamate-only docker image with hooks to existing Grafana / Postgres / MQTT services (and not duplicating the complete infrastructure), or (preferably) a manual installation option.

Thank you!

### Expected Behavior

-

### Steps To Reproduce

-

### Relevant log output

```shell
-
```

### Screenshots

-

### Additional data

-

### Type of installation

Manual

### Version

1.28

**Comment by greggitter** [comment]
Is there a risk of manual install going away?  My setup is manual debian.  Thx.

**Comment by brianmay** [comment]
My understanding is that the discussion was concerning the future of the documentation. Manual installs would continue to work even if the documentation is removed.

I don't think anybody was going to actually remove the documentation.

I think there was frustration that we have to keep multiple installation procedures tested and in sync. I don't think that is a good reason for removing documentation though.

If anything, I would like to see more installation methods documented, e.g. [nix](https://github.com/teslamate-org/teslamate/pull/3485).

**Comment by cwanja** [comment]
I am going to move this to a discussion for now, as I do not see an "issue" at the moment.

**Reply by greggitter** [reply]
Well this is embarrassing....after posting this I found the file I needed.  To resolve this I went here https://www.erlang-solutions.com/downloads/# and downloaded the latest file (26.1.1-1) from debian, and used dpkg to install.

Thanks for all the work you guys/gals (?) put in!

**** Resolved ****

Hi, this is sort of related I think to the discussion about manual install method and documentation.  As of this latest version the compiling no longer works as it looks like the new requirement is Erlang v26 though I can't find confirmation on this.  I guess that would be my first question.  It doesn't look as simple as using apt to install a new version since the packaged version is v25 with debian 12.  What I'm looking for is a recommended way to download the new version (compile from source, use asdf or rabbitmq, etc), and specifically which version so the compiles will work again.  Here's the error I'm getting:


```
root@ct-teslamate:/usr/src/teslamate # mix deps.get --only prod
=ERROR REPORT==== 27-Oct-2024::13:52:21.612737 ===
beam/beam_load.c(184): Error loading module 'Elixir.Kernel':
  This BEAM file was compiled for a later version of the runtime system than the current (Erlang/OTP 25).
  To fix this, please re-compile this module with an Erlang/OTP 25 compiler.
  (Use of opcode 182; this emulator supports only up to 180.)


=ERROR REPORT==== 27-Oct-2024::13:52:21.612780 ===
Loading of /usr/lib/elixir/bin/../lib/elixir/ebin/Elixir.Kernel.beam failed: badfile

=ERROR REPORT==== 27-Oct-2024::13:52:21.627177 ===
Loading of /usr/lib/elixir/bin/../lib/elixir/ebin/elixir_code_server.beam failed: badfile

=ERROR REPORT==== 27-Oct-2024::13:52:21.627167 ===
beam/beam_load.c(184): Error loading module elixir_code_server:
  This BEAM file was compiled for a later version of the runtime system than the current (Erlang/OTP 25).
  To fix this, please re-compile this module with an Erlang/OTP 25 compiler.
  (Use of opcode 181; this emulator supports only up to 180.)
```

Thanks...and again, just looking for a way forward.  Much thanks.
